api/tests/routes/backoffice/operations_test.py

```diff
@@ -628,3 +628,54 @@ def test_set_end_import_date(self, authenticated_client):
             .one()
         )
         assert db_response.endImportDate == date
+
+
+class UpdateVenueTest(PostEndpointHelper):
+    endpoint = "backoffice_web.operations.update_venue"
+    endpoint_kwargs = {"special_event_id": 1}
+    needed_permission = perm_models.Permissions.MANAGE_SPECIAL_EVENTS
+
+    def test_set_end_venue(self, authenticated_client):
+        event = operations_factories.SpecialEventFactory(venue=offerers_factories.VenueFactory())
+        venue = offerers_factories.VenueFactory()
+
+        response = self.post_to_endpoint(
+            authenticated_client,
+            special_event_id=event.id,
+            form={"venue": [f"{venue.id}"]},
+        )
+
+        assert response.status_code == 303
+
+        db_response = (
+            db.session.query(
+                operations_models.SpecialEvent.venueId,
+            )
+            .filter(
+                operations_models.SpecialEvent.id == event.id,
+            )
+            .one()
+        )
+        assert db_response.venueId == venue.id
```

Comment:
Suggestion : Pourquoi pas juste 
```suggestion
        db.session.refresh(event)
        assert event.venue == venue
```
?  (ce qui rend la compréhension bien plus immédiate)

Et pareil avec `event.venue is None` dans le test en-dessous.